src/lib/Libattr/master_job_attr_def.xml

```diff
@@ -2054,6 +2054,23 @@
          <ECL>NULL_VERIFY_VALUE_FUNC</ECL>
       </member_verify_function>
    </attributes>
+   <attributes>
+      <member_index>JOB_ATR_max_run_subjobs</member_index>
+      <member_name>ATTR_max_run_subjobs</member_name>
+      <member_at_decode>decode_l</member_at_decode>
+      <member_at_encode>encode_l</member_at_encode>
+      <member_at_set>set_l</member_at_set>
+      <member_at_comp>comp_l</member_at_comp>
+      <member_at_free>free_null</member_at_free>
+      <member_at_action>NULL_FUNC</member_at_action>
+      <member_at_flags>READ_WRITE | ATR_DFLAG_ALTRUN</member_at_flags>
+      <member_at_type>ATR_TYPE_LONG</member_at_type>
+      <member_at_parent>PARENT_TYPE_JOB</member_at_parent>
+      <member_verify_function>
+         <ECL>verify_datatype_long</ECL>
+         <ECL>verify_value_zero_or_positive</ECL>
```

Comment:
It's zero or positive because if someone wants to hold off running subjobs without penalizing the accrue type or its position in the calendar then they can do so.
I'll add this to the design doc as well.